docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+
+**Converters**
+
+- `Converters`_
+- `Evolutions`_
+- `Expectations`_
+
+**Gradients**
+
+- `Gradients`_
+
+
+Operator Base Class
+===================
+*Back to* `Contents`_
+
+.. |qiskit.quantum_info.BaseOperator| replace:: ``qiskit.quantum_info.BaseOperator``
+.. _qiskit.quantum_info.BaseOperator: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/quantum_info/operators/base_operator.py
+
+The :class:`qiskit.opflow.OperatorBase` abstract class can be replaced with |qiskit.quantum_info.BaseOperator|_ ,
+keeping in mind that |qiskit.quantum_info.BaseOperator|_ is more generic than its opflow counterpart.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+   * - :class:`qiskit.opflow.OperatorBase`
+     - |qiskit.quantum_info.BaseOperator|_
+
+..  attention::
+
+    Despite the similar class names, :class:`qiskit.opflow.OperatorBase` and
+    |qiskit.quantum_info.BaseOperator|_ are not completely equivalent to each other, and the transition
+    should be handled with care. Namely:
+
+    1. :class:`qiskit.opflow.OperatorBase` implements a broader algebra mixin. Some operator overloads that were
+    commonly used :mod:`~qiskit.opflow` (for example ``~`` for ``.adjoint()``) are not defined for
+    |qiskit.quantum_info.BaseOperator|_. You might want to check the specific
+    :mod:`~qiskit.quantum_info` subclass instead.
+
+    2. :class:`qiskit.opflow.OperatorBase` also implements methods such as ``.to_matrix()`` or ``.to_spmatrix()``,
+    which are only found in some of the |qiskit.quantum_info.BaseOperator|_ subclasses.
+
+    See API reference for more information.
+
+
+Operator Globals
+================
+*Back to* `Contents`_
+
+Opflow provided shortcuts to define common single qubit states, operators, and non-parametrized gates in the
+|operator_globals|_ module.
+
+These were mainly used for didactic purposes or quick prototyping, and can easily be replaced by their corresponding
+:mod:`~qiskit.quantum_info` class: :class:`~qiskit.quantum_info.Pauli`, :class:`~qiskit.quantum_info.Clifford` or
+:class:`~qiskit.quantum_info.Statevector`.
+
+
+1-Qubit Paulis
+--------------
+*Back to* `Contents`_
+
+The 1-qubit paulis were commonly used for quick testing of algorithms, as they could be combined to create more complex operators
+(for example, ``0.39 * (I ^ Z) + 0.5 * (X ^ X)``).
+These operations implicitly created operators of type  :class:`~qiskit.opflow.primitive_ops.PauliSumOp`, and can be replaced by
+directly creating a corresponding :class:`~qiskit.quantum_info.SparsePauliOp`, as shown in the examples below.
+
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow
+     - Alternative
+   * - :class:`~qiskit.opflow.X`, :class:`~qiskit.opflow.Y`, :class:`~qiskit.opflow.Z`, :class:`~qiskit.opflow.I`
+     - :class:`~qiskit.quantum_info.Pauli`
+
+       ..  tip::
+
+           For direct compatibility with classes in :mod:`~qiskit.algorithms`, wrap in :class:`~qiskit.quantum_info.SparsePauliOp`.
```

Comment:
It's a good tip. Can you give an example here. Do you mean something like `X = SparsePauliOp(Pauli('X'))`?

Reply:
By qiskit.algorithms this really means the new primitive based ones there right. Arguably, like the Estimator they used, they take BaseOperator as type which should mean they (and the Estimator they use) support Pauli directly.

Reply:
I wonder with these tables similar to this outputs (as per first image below) whether a vertical top align, rather than center, would be better. It would align the two text blocks we most want to compare side by side and leave the tip (or note in other cases) lower down.

![image](https://user-images.githubusercontent.com/40241007/222277452-d0efd99b-7186-4096-a409-bcd99ff88447.png)

So it would look more like this (I edited the above image)

![image](https://user-images.githubusercontent.com/40241007/222277974-826ca30d-5cb4-459a-a458-96be67fcfa2f.png)

Reply:
@woodsp-ibm I agree that this looks nicer, but I think this is more of a sphinx theme issue in general, right? Maybe better suited for this issue?? https://github.com/Qiskit/qiskit_sphinx_theme/issues/193

Reply:
I had not looked at the source to see what you used. Normally cell alignment is a choice - maybe what was used gives no choice for the outcome. I made a similar comment as above in that sphinx theme issue you linked.